docs/project/contribution_tools.md

```diff
@@ -71,6 +72,16 @@ pre-commit install
 ./scripts/run_bazelisk.py test //...:all
 ```
 
+### Configuring `clangd`
+
+To ensure that `clangd` reports accurate diagnostics it
+needs a generated file called `compile_commands.json`, this can be generated by
+invoking the command below.
```

Comment:
```suggestion
To ensure that `clangd` reports accurate diagnostics, it
needs a generated file called `compile_commands.json`. This can be generated by
invoking:
```

Sorry, just some small phrasing nits.